github/table_github_audit_log.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v45/github"
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGitHubAuditLog(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_audit_log",
+		Description: "Gets the audit log for an organization.",
```

Comment:
```suggestion
		Description: "Gets the audit logs for an organization.",
```

Minor change, but plural reads a bit better.